types/react-relay/v1/classic.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for react-relay/classic 1.3
```

Comment:
Did you re-add these v1 files on purpose? I don’t think we really need them, it’s not like Relay 0.x is going to get any further updates afaik.

Reply:
@alloy sorry, just saw this _after_ merge. @uniqueiniquity had requested that I follow the patterns for updating major versions, which requires maintaining the previously existing types. https://github.com/DefinitelyTyped/DefinitelyTyped/pull/19533#pullrequestreview-63426763

Reply:
I see. OTOH @andy-ms notes that [we can delete them](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/19533#discussion_r144883373) if if we’re not going to update them anymore, which is the case. People using e.g. Relay 0.10 can just pin to the corresponding `@types/react-relay` package.

Reply:
Anyways, I’m fine with it if you wanna re-add them, but I think that’s the change that’s causing CI to fail atm, although its output is sparse to say the least.

Reply:
Hey, if we can delete them, I'm all for it. @uniqueiniquity could you weigh in here?

Reply:
Either way, shouldn’t this be `v0` instead of `v1`?

Reply:
Go ahead with @andy-ms's recommendation. If they won't be updated in the future, then it's fine to delete them.